content/en/blog/2023/announcing-notation-rc6.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Notation v1.0.0-RC.7 is available!
+author:  "Notation Release Team"
+date: 2023-05-28
+draft: false
+---
+
+The Notation maintainers are pleased to announce the release of Notation v1.0.0-RC.7, including [Notation CLI v1.0.0-rc.7](https://github.com/notaryproject/notation/releases/tag/v1.0.0-rc.7), [notation-go library v1.0.0-rc.6](https://github.com/notaryproject/notation-go/releases/tag/v1.0.0-rc.6), and [notation-core-go library v1.0.0-rc.4](https://github.com/notaryproject/notation-core-go/releases/tag/v1.0.0-rc.4). This blog walks you through the updates in this release.
+
+> NOTE: Both Notation CLI v1.0.0-rc.7 and v1.0.0-rc.6 have the same functionalities. However, v1.0.0-rc.7 included an additional fix for an E2E test case.
+
+## What's new
+
+This release adds the following major changes:
+
+- Security advisory fixes
+- Notation commands support reading Docker credentials if the [credentials store](https://docs.docker.com/engine/reference/commandline/login/#configure-the-credentials-store) is not configured
+- Renamed `--plain-http` to `--insecure-registry`, which should be used only for testing
+
+Other changes:
+
+- Improved error messages
+- Bug fixes
+- Updated dependencies
+
+### Notation commands support reading docker credentials if credentials store is not installed
```

Comment:
Should it be more appropriate to say, 
Notation commands support reading docker credentials when an explicit credential store is not configured in Notation

Reply:
Dont we always need a credential store?

Reply:
Storing credentials in plaintext is bad. However, we should not stop notation **reading** those existing plaintext credentials.

Reply:
@priteshbandi @iamsamirzon Notation never store plain text credentials, but can read plain text credentials from docker config

Reply:
Agree, what is available in Docker Config is a method we can use. But can someone store plaintext credentials in Notation's config.json?